scripts/categorizer.py

```diff
@@ -109,6 +112,30 @@ def determine_if_function_def(line_number, multiline_number, line, categorizatio
 
     return 0
 
+def determine_if_function_call(line_number, multiline_number, line, categorizations):
+    """
+    Determines if the line has a function associated with it.
+
+    :param line_number:
+    :param multiline_number:
+    :param line:
+    :param categorizations:
+    :return:
+    """
+    if "def" not in line or re.search("def\s", line.strip(" ")[0:4]) == None:
```

Comment:
Comments? like # Checking whether this function is a function def, and if it is ignoring it?